docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -206,8 +206,7 @@ Most modern browsers support the ability to "lazy load" images by adding a `load
 to the `<img />` tag. This defers the loading of images not in the viewport to improve the initial 
 page load performance.
 
-Silverstripe CMS automatically adds the `loading="lazy"` to images added in an HTML editor field
-and to images rendered via a SS template file.
+As of SilverStripe CMS 4.9, the `loading="lazy"` attribute is automatically added to images added in an HTML editor field and to images rendered via a SS template file.
```

Comment:
If we do that systematically, all our doc articles are going to be a mess of "available as-of Silverstripe CMS xyz".

I fully appreciate that this is not ideal, but I think writting our doc with the assumption that people are using the latest release is the lesser evil.

Reply:
Okie dokie